_posts/2020-08-04-hhvm-4.69.markdown

```diff
@@ -0,0 +1,22 @@
+---
+title: "HHVM 4.69"
+layout: post
+author: fred
+category: blog
+---
+
+HHVM 4.69 is released! This release marks the end of support for 4.62;
+HHVM 4.63&ndash;4.68 remain supported, as do the 4.32 and 4.56 LTS releases.
+
+# Breaking Changes
+
+- Directly implementing the same interface twice with multiple generics is now
+  banned, due to incorrect over-permissiveness.
```

Comment:
I don't quite understand this one, an example could be useful.

Reply:
will update - `class foo implements I<string>, I<int> {}` would only properly enforce `I<string>`